themes/src/main/resources/theme/base/login/webauthn-authenticate.ftl

```diff
@@ -30,7 +30,7 @@
 
                         <div class="${properties.kcFormClass!}">
                             <#list authenticators.authenticators as authenticator>
-                                <div id="kc-webauthn-authenticator" class="${properties.kcSelectAuthListItemClass!}">
+                                <button id="kc-webauthn-authenticator" class="${properties.kcSelectAuthListItemClass!}">
```

Comment:
Just realized these `id` attributes are likely not unique, as this is inside of a loop. We don't have to fix this now, so I logged a separate issue for it (#24462).